app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,21 @@
+<footer class="footer">
+  <small>
+    <%= t 'matched.the' %>
+    <a href="http://www.railstutorial.org/">
+      <%= t 'common.rubyonrails_tut' %>
+    </a>
+    <%= t 'matched.by' %>
+    <a href="http://www.michaelhartl.com/">
+      <%= t 'common.rails_book_author' %>
+    </a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "#{t 'title.about'}", about_path %></li>
```

Comment:
Lúc em đổi sang như thế thì có lỗi này xuất hiện ạ:
undefined method `has_key?' for "/about":String